tests/unit/src/main/scala/tests/BaseRenameLspSuite.scala

```diff
@@ -124,9 +125,12 @@ abstract class BaseRenameLspSuite(name: String) extends BaseLspSuite(name) {
                     Future
                       .sequence {
                         files.map { case (file, code) =>
-                          server.didSave(file)(_ =>
-                            code.replaceAll(allMarkersRegex, "")
-                          )
+                          for {
+                            _ <- server.didChange(file)(_ =>
+                              code.replaceAll(allMarkersRegex, "")
+                            )
+                            _ <- server.didSave(file)(identity)
```

Comment:
Maybe we should do either of two:
 - remove change function from `server.didChange` (it will always de identity)
 - have `server.didSave` with the function send first `didChange` and only later `didSave`.

Reply:
> have server.didSave with the function send first didChange and only later didSave.

That proved to be complicated and broke a few things, I will change to didSave always doing identity

Reply:
Remove fn from didSave to make it explicit that it doesn't actually send anything. And also this wasn't simulating correctly how the editor would work